lib/apia/open_api/objects/parameters.rb

```diff
@@ -87,14 +103,31 @@ def generate_argument_set_params
             next if child_arg.type.argument_set?
 
             param = {
-              name: "#{@argument.name}[#{child_arg.name}]",
-              in: "query",
-              schema: generate_scalar_schema(child_arg)
+              in: "query"
             }
+
             description = []
             description << formatted_description(@argument.description) if @argument.description.present?
             description << formatted_description(child_arg.description) if child_arg.description.present?
-            description << "All '#{@argument.name}[]' params are mutually exclusive, only one can be provided."
+
+            if @argument.type.id.end_with?("Lookup")
+
+              description << "\n\nAll '#{@argument.name}[]' params are mutually exclusive, only one can be provided."
+
+              # else
+              #   require "pry-remote"
+              #   binding.remote_pry
+            end
+
+            if @argument.array
```

Comment:
It would be good to add some testing for this so we can verify the output against the fixtures

Reply:
New tests or updating the example to use an array of argument sets ? 
I've now done the latter and the text fixture is updated as expected :bow: